hr_employee_id/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Employee Identification Numbers
```

Comment:
OK, don't worry. We are here to help. I mean to copy the same line `=====` at the top. This is a very recent addition, because we were not doing well the styles. Look at this PR to understand it: https://github.com/OCA/maintainer-tools/pull/109